packages/yarnpkg-builder/package.json

```diff
@@ -24,6 +24,7 @@
     "webpack-sources": "^1.4.3"
   },
   "devDependencies": {
+    "@datadog/build-plugin": "0.3.3",
```

Comment:
Needs to be a normal dependency, or we need to modify the builder so we only have this enabled when run in the berry repo